pkg/domain/infra/abi/play.go

```diff
@@ -1511,7 +1511,7 @@ func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, body io.Reader, opt
 		return nil, err
 	}
 
-	reports.RmReport, err = ic.PodRm(ctx, podNames, entities.PodRmOptions{Ignore: true})
+	reports.RmReport, err = ic.PodRm(ctx, podNames, entities.PodRmOptions{Ignore: true, Force: true})
```

Comment:
That only sets `pod rm --force` which will stop running containers instead of erroring out.

Reply:
I believe it will also push the force to remove the containers as well.

Reply:
But if I can get the buildah test to work, then we can actually see if it works.

Reply:
I did not test it yet but my assumption is that this is not enough for the Buildah container.  Imagine only one container exists ... the pod won't exist, so there's is nothing to remove.  I'd expect the container to not be removed then.  But I may be off.

Reply:
Added tests using your buildah idea.